src/Defender/Traits/Models/Permission.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Artesaos\Defender\Traits\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Artesaos\Defender\Pivots\PermissionRolePivot;
+use Artesaos\Defender\Pivots\PermissionUserPivot;
+
+/**
+ * Trait Permission.
+ */
+trait Permission
+{
+    /**
+     * @var
+     */
+    protected $table;
+
+    /**
+     * @param array $attributes
+     */
+    public function __construct(array $attributes = [])
+    {
+        // Must to be declared before parent::__construct call
+        $this->fillable = $fillable = [
```

Comment:
It would be a best possible merge with the own user attributes.
Anyway I do not know how far this part is productive